src/lib.rs

```diff
@@ -701,6 +701,53 @@ impl Format {
         }
     }
 
+    /// Conservative bounds for number of bytes read while attempting to match a format
```

Comment:
Maybe we could say "conservative over-estimate" to make it clear which direction it's being conservative in.